addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -710,7 +710,6 @@
             <field name="amount_type">percent</field>
             <field name="amount">19</field>
             <field name="type_tax_use">sale</field>
-            <field name="price_include">False</field>
```

Comment:
Maybe the intention of this one was to do `eval="False"`. It was evaluated as `True` so your commit won't change but cc @jco-odoo in case he wants to modify the line below to `eval="False"`
same comment below and in l10n_de_skr04